tests/sort-directory-order-test/Makefile

```
@@ -1,7 +1,7 @@
-test: clean lines1.output
+test: lines1.output
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)